go.mod

```diff
@@ -143,11 +147,10 @@ require (
 	k8s.io/api v0.24.1 // indirect
 )
 
-require github.com/gofrs/uuid v4.2.0+incompatible // indirect
-
 replace (
-	github.com/docker/cli => github.com/docker/cli v20.10.3-0.20220309205733-2b52f62e9627+incompatible
-	github.com/docker/docker => github.com/docker/docker v20.10.3-0.20220309172631-83b51522df43+incompatible
+	github.com/docker/cli => github.com/docker/cli v20.10.3-0.20221013132413-1d6c6e2367e2+incompatible // 22.06 master branch
+	github.com/docker/docker => github.com/docker/docker v20.10.3-0.20221013203545-33ab36d6b304+incompatible // 22.06 branch
+	github.com/moby/buildkit => github.com/moby/buildkit v0.10.1-0.20220816171719-55ba9d14360a // same as buildx
 
```

Comment:
oh! compose is already using 22.06 branch? Hm...

Reply:
yes, it was needed to use latest features offered by buildx (which is doing the same btw)